benchmarks/python/test_rmsnorm_add_fwd.py

```diff
@@ -30,3 +70,35 @@ def test_rmsnorm_add_fwd_baseline_benchmark(
         benchmark_fn,
         [inputs, weights, residual],
     )
+
+
+# flash infer does inplace update of inputs and residual tensors
+# needs to explicitly return the updated tensors to correctly compute IO bytes
+# https://github.com/flashinfer-ai/flashinfer/blob/ba2b4aa636c4ecf99981794767ffbf89267720cd/flashinfer/norm.py#L106
+def flashinfer_rmsnorm_add_wrapper(inputs_list):
+    inputs, weights, residual = inputs_list
+    fused_add_rmsnorm(inputs, residual, weights)
+    return inputs, residual
```

Comment:
Alternatively, you can pass computed iobytes to `run_benchmark`. Not a requirement, you can choose what seems most viable.